src/lib/apiService.ts

```diff
@@ -167,11 +198,8 @@ const getTransactions = async (startDate?: Moment, endDate?: Moment): Promise<FF
             queryString += `${queryString.length > 0 ? '&' : '?'}end=${endDate.format(DATE_FORMAT)}`;
         }
     }
-    const response = await http?.get(`/transactions${queryString}`, exceptionHandling);
-    if (response && response.status === 200 && response.data.data) {
-      return response.data.data;
-    }
-    return [];
+    const transactions = await Array.fromAsync(pageThroughResource(http, `transactions${queryString}`));
+    return transactions;
```

Comment:
Transactions needs to be cast here:
`return transactions as FF3Wrapper<FF3Transaction>[];`